core/src/main/scala/akka/kafka/KafkaConsumerActor.scala

```diff
@@ -181,6 +185,34 @@ private[kafka] class KafkaConsumerActor[K, V](settings: ConsumerSettings[K, V])
       else consumer.pause(java.util.Collections.singleton(tp))
     }
 
+    def tryPoll(timeout: Long, onSuccess: ConsumerRecords[K, V] => Unit): Unit = {
```

Comment:
I'll change this block to ordinary `try-catch` to avoid unnecessary allocations.